src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj

```diff
@@ -7,7 +7,7 @@
     <AssemblyName>Microsoft.Azure.Management.Batch</AssemblyName>
     <VersionPrefix>5.1.0</VersionPrefix>
     <PackageTags>Microsoft Azure batch management</PackageTags>
-    <PackageReleaseNotes>For detailed release notes, see: https://github.com/Azure/azure-sdk-for-net/blob/vs17Dev/src/SDKs/Batch/Management/changelog.md</PackageReleaseNotes>
+    <PackageReleaseNotes>For detailed release notes, see: https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Batch/Management/changelog.md</PackageReleaseNotes>
```

Comment:
@matthchr as long as you know this will not work when psSdkJson6 is collapsed to vs17Dev
Another option is to use aka.ms url